README.adoc

```diff
@@ -484,3 +479,21 @@ NOTE: If I can figure out a good way to setup a Ruby testing environment I'll do
     ...
 </configuration>
 -----
+
+=== Add version and build date to the header
+
+Automatically add version details to header and footer to all documents.
+
+[source, xml]
+-----
+<configuration>
+    ...
+    <attributes>
+        ...
+        <revnumber>${project.version}</revnumber>
+        <revdate>${maven.build.date}</revdate>
```

Comment:
This should actually be maven.build.timestamp...or we should explain how to get Maven to expose the build date. Maybe even suggest integrating with the Maven git plugin. Otherwise, people are going to ask by maven.build.date doesn't work.